uber/models.py

```diff
@@ -630,9 +630,22 @@ def everything(self, location=None):
                                         .order_by(Attendee.full_name).all()
                          if c.AT_THE_CON or not location or int(location) in a.assigned_depts_ints]
             for job in jobs:
-                job._available_staffers = [a for a in attendees if not job.restricted or a.trusted]
+                job._available_staffers = [a for a in attendees if not job.restricted or a.trusted_in(job.location)]
             return jobs, shifts, attendees
 
```

Comment:
I just did some searching and I can't find out why this is here, it's probably cruft.  we can probably just remove it because 1) it's not used and 2) .available_staffers pretty much does the same exact thing if it was ever needed again.